Cargo.lock

```diff
@@ -220,7 +222,7 @@ checksum = "a06f77d526c1a601b7c4cdd98f54b5eaabffc14d5f2f0296febdc7f357c6d3ba"
 
 [[package]]
 name = "gapstat-rs"
-version = "2.0.0-rc2"
+version = "2.0.1"
```

Comment:
I'm a bit confused why changes in `Cargo.lock`? Not relevant to the changes in python files.

Reply:
Ah, I think I updated that file before I decided that setting the rng seed for k-means would go beyond what is currently possible with sklearn. Reverted.